### PR TITLE
Python Code Quality

### PR DESCRIPTION
**Pull request by chuyskywalker**
Tons of little things; see the commit list since I tried to make each one self-explanitory

## Motivation and Context
Clean up the codebase to be more consistent and apply best practices

## How Has This Been Tested?
Ran it.

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Comment by Ashex** [comment]
Why did you switch all the string formatters to the old style? It's good that it's now consistent but it seems an odd choice to use the old instead of new.


https://pyformat.info/#simple

**Comment by chuyskywalker** [comment]
It's not actually the old style, that would be:

```
log.info("string %s" % variable)
```

This is
```
log.info("string %s", variable)
```

It's a very subtle difference, but the theory is that `variable` resolution is deferred in case that log level isn't actually rendered. That way you're don't doing a whole bunch of `.format()` calls when you're not actually logging the data.

**Comment by Ashex** [comment]
I see now, I completely missed that these changed were only to logging. 

https://docs.python.org/3/howto/logging.html#logging-variable-data

**Comment by AHAAAAAAA** [comment]
rebase. +1 for merge

File: pogom/models.py [hunk 1]
```diff
@@ -279,60 +279,57 @@ def parse_map(map_dict, iteration_num, step, step_location):
...
         bulk_upsert(Pokemon, pokemons)
 
     if pokestops and config['parse_pokestops']:
         pokestops_upserted = len(pokestops)
-        log.debug("Upserting {} pokestops".format(len(pokestops)))
         bulk_upsert(Pokestop, pokestops)
 
     if gyms and config['parse_gyms']:
         gyms_upserted = len(gyms)
-        log.debug("Upserting {} gyms".format(len(gyms)))
```
Comment (Chlodochar) [inline pogom/models.py]: Why remove these? It seems confusing when Pokemon's are logged.

**Comment by chuyskywalker** [comment]
Kinda wish landscape.io would fucking work for this branch...

**Comment by Chlodochar** [comment]
OT, but we might want to switch to SideCI.

**Comment by chuyskywalker** [comment]
As long as you can turn off it's "here, let me leave you 5000 github notifications" feature ;D

**Comment by Chlodochar** [comment]
If the current problems are solved it seems alright to have a comment for each new problem.

**Pull request merged**